assets/queries/k8s/k8s_dashboard_integration/query.rego

```diff
@@ -0,0 +1,82 @@
+package Cx
+
+CxPolicy [ result ] {
+   document := input.document
+   metadata := document[i].metadata
+   kind := document[i].kind
+   check_kind(kind)
+   
+   labels := metadata.labels
+   label_name := "k8s-app"
+   labels[label_name] == "kubernetes-dashboard"
+   
+   result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":      sprintf("metadata.name=%s", [metadata.name]),
+                "issueType":		   "IncorrectValue",
+                "keyExpectedValue": sprintf("metadata.name=%s.labels[%s] has not kubernetes-dashboard deployed", [metadata.name, label_name]),
+                "keyActualValue": 	sprintf("metadata.name=%s.labels[%s] has kubernetes-dashboard deployed", [metadata.name, label_name])
+              }
+}
+
+CxPolicy [ result ] {
+   document := input.document
+   metadata := document[i].metadata
+   kind := document[i].kind
+   check_kind(kind)
+   
+   rules := document[i].rules[_].resourceNames[_]
+   
+   contains(rules, "kubernetes-dashboard")
+   
+   result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":      sprintf("metadata.name=%s.rules", [metadata.name]),
+                "issueType":		   "IncorrectValue",
+                "keyExpectedValue": sprintf("None of the metadata.name=%s.rules has kubernetes-dashboard deployed", [metadata.name]),
+                "keyActualValue": 	sprintf("One of the metadata.name=%s.rules has kubernetes-dashboard deployed", [metadata.name])
+              }
+}
+
+CxPolicy [ result ] {
+   document := input.document
+   metadata := document[i].metadata
+   kind := document[i].kind
+   check_kind(kind)
+   
+   roleRef := document[i].roleRef.name
+   
+   contains(roleRef, "kubernetes-dashboard")
+   
+   result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":      sprintf("metadata.name=%s.roleRef", [metadata.name]),
+                "issueType":		   "IncorrectValue",
+                "keyExpectedValue": sprintf("metadata.name=%s.roleRef has kubernetes-dashboard deployed", [metadata.name]),
+                "keyActualValue": 	sprintf("metadata.name=%s.roleRef has kubernetes-dashboard deployed", [metadata.name])
+              }
+}
```

Comment:
I think a better approach would be searching for containers, inside the "containers" or "initContainers" list, whose "image" value contains "kubernetes-dashboard" or "kubernetesui".